css/style.css

```diff
@@ -1035,5 +1037,127 @@ ul {
 /* Faq End */
 
 /* Hero start */
-
-/* Hero End */
\ No newline at end of file
+.hero{
+  width: 90%;
+  margin: auto;
+  max-width: 1500px;
+  user-select: none;

```

Comment:
padding: 110px 0;

add padding in hero section and make responsive (reduce the size in phone and tablet section)